test-requirements.txt

```diff
@@ -1,3 +1,4 @@
 nose
 mock
 flask<0.12
+django<1.11
```

Comment:
Yea, I'd like to see this changed to `Django>=1.8` since 1.8 is the oldest supported Django.

I'm working on a PR to add a tox build that will run the tests against the Python 2/3 and Django version combinations.